cypress/integration/workflows.spec.js

```diff
@@ -33,7 +33,7 @@ context('Workflows', () => {
       .find('[data-cy=workflow-row]')
       .should('contain.text', 'wf_open')
       .should('not.contain.text', 'wf_timedout')
-      .should('not.contain.text', 'wf_terminated')
+      .should('not.contain.text', 'wf_terminated1')
```

Comment:
where does this come from?

Reply:
in our CI it comes from the cicd project, particularly `run-tests-web.yml`